crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -814,6 +848,16 @@ type Owner implements ObjectOwner {
 	"""
 	stakedSuiConnection(first: Int, after: String, last: Int, before: String): StakedSuiConnection
 	defaultNameServiceName: String
+	"""
```

Comment:
Clarify in these docs that this API is useful if you are trying to access a dynamic field of a wrapped object? (e.g. in addition to the updated docs I suggested in the comment above).